src/main/java/book/store/onlinebookstore/model/User.java

```diff
@@ -55,6 +57,11 @@ public class User implements UserDetails {
     )
     private Set<Role> roles = new HashSet<>();
 
+    @ToStringExclude

```

Comment:
what is this annotation?

`org.apache.commons.lang3.builder.ToStringExclude` - will it work with Lombok?